style.css

```diff
@@ -2,579 +2,1338 @@
+  }
 }
 
-.contact-social-icons hr {
-  width: 134px;
-  height: 5px;
-  margin: 23px 0 8px 50px;
-  background: #0d151e;
-  border-radius: 10px;
+@media (min-width: 760px) {
+  * {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+
+  .moble-container {
+    display: none;
+  }
+
+  .desktop-nav-container {
+    display: flex;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    justify-content: flex-end;
+    align-items: center;
+    position: fixed;
+    overflow: hidden;
+    background-color: #3e3c3c;
+  }
+
+  .desklogo {
+    width: 120px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+    cursor: pointer;
+    text-decoration: none;
+    margin-right: auto;
+    margin-left: 100px;
+  }
+
+  .nav-links a {
+    transition: all 0.3s ease 0s;
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    color: #fff;
+  }
+
+  .nav-links {
+    display: flex;
+    list-style: none;
+    padding: 0;
+  }
+
+  .nav-links li {
+    padding: 26px 0 26px 32px;
+  }
+
+  .nav-links img {
+    margin-right: 50px;
+  }
+
+  .desktop-headline {
+    display: flex;
+    min-height: 870px;
+    background-image: url(images/Illustration-header-desk.png);
+    background-color: #1c1a19;
+    background-repeat: no-repeat;
+    background-size: 30% auto;
+    background-position: right;
+  }
+
+  .desk-social-icons ul {
+    list-style: none;
+    margin-top: 279px;
+  }
+
+  .desk-social-icons li {
+    margin-top: 30px;
+    margin-left: 40px;
+  }
+
+  .desk-intro {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .desk-intro h1 {
+    width: 566px;
+    height: 160px;
+    margin: 192px 0 0 88px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+  }
+
+  .desk-intro h2 {
+    width: 566px;
+    height: 24px;
+    margin: 24px 0 0 88px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    align-items: center;
+    color: #fff;
+  }
+
+  .desk-intro p {
+    width: 566px;
+    height: 123px;
+    margin: 24px 0 0 88px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  .recent-heading {
+    display: flex;
+  }
+
+  .recent-heading h1 {
+    width: 315px;
+    height: 52px;
+    margin: 135px 0 0 153px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    align-items: center;
+    color: #091e42;
+  }
```

Comment:
Kindly check and make sure that the widht and height of the title conforms with the design

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-12 12-18-19](https://user-images.githubusercontent.com/65233860/201474237-35fa0d4c-dec1-4bb0-b382-01043bed7f78.png)  |  ![Screenshot from 2022-11-12 12-19-07](https://user-images.githubusercontent.com/65233860/201474244-5a0abe8c-a0a1-45ac-b18b-1d2a1d6af85d.png)